sql/information_schema/information_schema.go

```diff
@@ -947,6 +961,39 @@ func innoDBTempTableIter(ctx *Context, c Catalog) (RowIter, error) {
 	return RowsToRowIter(rows...), nil
 }
 
+func processListIter(ctx *Context, c Catalog) (RowIter, error) {
+	processes := ctx.ProcessList.Processes()
+	var rows = make([]Row, len(processes))
+
+	db := ctx.GetCurrentDatabase()
+	if db == "" {
+		db = "NULL"
+	}
+
+	for i, proc := range processes {
+		var status []string
+		for name, progress := range proc.Progress {
+			status = append(status, fmt.Sprintf("%s(%s)", name, progress))
+		}
+		if len(status) == 0 {
+			status = []string{"running"}
```

Comment:
running or executing?

Reply:
I made it compatible to result of current `SHOW PROCESSLIST` query, which any process state defaults to 'running' for now. But it should have multiple options like 'running', 'executing', 'starting', etc.